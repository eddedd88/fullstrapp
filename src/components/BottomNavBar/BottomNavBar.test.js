import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import BottomNavBarWithRouter, { BottomNavBar } from './BottomNavBar'

describe('BottomNavBar', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(
      <BottomNavBar
        classes={{ test: 'test' }}
        location={{
          pathname: '/reasons',
          hash: '',
          search: ''
        }}
      />
    )
    expect(tree).toMatchSnapshot()
  })

  it('renders with router correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(<BottomNavBarWithRouter />)
    expect(tree).toMatchSnapshot()
  })
})
