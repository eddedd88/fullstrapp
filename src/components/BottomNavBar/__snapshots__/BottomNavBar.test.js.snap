// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomNavBar renders correctly 1`] = `
<WithStyles(BottomNavigation)
  classes={
    Object {
      "test": "test",
    }
  }
  showLabels={true}
  value="/reasons"
>
  <WithStyles(BottomNavigationAction)
    component={[Function]}
    icon="div"
    label="Feed"
    to="/feed"
    value="/feed"
  />
  <WithStyles(BottomNavigationAction)
    component={[Function]}
    icon="div"
    label="Grid"
    to="/grid"
    value="/grid"
  />
  <WithStyles(BottomNavigationAction)
    component={[Function]}
    icon="div"
    label="Profile"
    to="/profile"
    value="/profile"
  />
</WithStyles(BottomNavigation)>
`;

exports[`BottomNavBar renders with router correctly 1`] = `
<Route>
  [Function]
</Route>
`;
